pkg/sdk/go/things_test.go

```diff
@@ -78,28 +79,28 @@ func TestCreateThing(t *testing.T) {
 	}{
 		{
 			desc:     "create new thing",
-			thing:    thing,
+			thing:    thing1,
 			token:    token,
 			err:      nil,
-			location: "001",
+			location: "fe6b4e92-cc98-425e-b0aa-000000000001",
 		},
 		{
 			desc:     "create new empty thing",
 			thing:    emptyThing,
 			token:    token,
 			err:      nil,
-			location: "002",
+			location: "123e4567-e89b-12d3-a456-000000000002",
```

Comment:
Same here, use `th2.ID`

Reply:
This is basically coming from pkg/uuid/mock.go.  So unlike above we can not use th2.ID, the test case would fail, if the value is changed.

Reply:
This could be done by using the Prefix const defined in the mock: `uuidMock.Prefix + "000000000002"` or as it's done in `pkg/uuid/mock``fmt.Sprintf("%s%012d", uuidMock.Prefix, 2)`

Reply:
Updated with location: fmt.Sprintf("%s%012d", uuid.Prefix, 2),